test/Constraints/overload.swift

```diff
@@ -92,7 +92,7 @@ func f4(_ j: Wibble) { } // expected-error{{cannot find type 'Wibble' in scope}}
 f4(5)
 
 func f1() {
-  var c : Class // expected-error{{cannot find type 'Class' in scope}}
+  var c : Klass // expected-error{{cannot find type 'Klass' in scope}}
   markUsed(c.x) // make sure error does not cascade here
```

Comment:
The intent of the test here is to avoid propagating the original error. Once this additional Clang module is brought in, the name `Class` somehow conflicts with name-resolution logic as affected by the newly-added implicit Clang module dependency. Renaming this variable allows us to keep testing the original thing and not worry about it being affected by brought-in clang declarations...

Reply:
Sounds good to me!